flex/flex-tests/testSrc/com/intellij/lang/javascript/FlexHighlightingTest.java

```diff
@@ -734,6 +734,11 @@ public void testConstructorNotAllowedInMxml() throws Exception {
     doTestFor(true, getTestName(false) + ".mxml", getTestName(false) + "_2.as");
   }
 
+  @JSTestOptions({JSTestOption.WithFlexSdk})
+  public void testNativeConstructorAllowedInMxml() throws Exception {
+    doTestFor(true, getTestName(false) + ".mxml", getTestName(false) + "_2.as");
```

Comment:
Second file is not needed for this test. I'll remove the file itself and reference to it myself.